app/models/manageiq/providers/openstack/inventory/collector/cloud_manager.rb

```diff
@@ -15,7 +15,7 @@ def availability_zones
 
   def cloud_services
     return @cloud_services if @cloud_services.any?
-    @cloud_services = compute_service.handled_list(:services, {}, openstack_admin?)
+    @cloud_services = uniques(compute_service.handled_list(:services, {}, openstack_admin?))
```

Comment:
we can ignore graph refresh, its indexes will not allow duplicate records to be passed to persistor